test/files/neg/t5365b.check

```diff
@@ -0,0 +1,31 @@
+t5365b.scala:2: warning: match may not be exhaustive.
+It would fail on the following inputs: None, Some(_)
+  def nonExhautiveIfWeAssumeGuardsTrueOrFalse(x: Option[Int]): Int = x match {
+                                                                     ^
+t5365b.scala:6: warning: match may not be exhaustive.
+It would fail on the following input: Some(_)
+  def nonExhautiveIfWeAssumeGuardsFalse(x: Option[Int]): Int = x match {
+                                                               ^
+t5365b.scala:11: warning: match may not be exhaustive.
+It would fail on the following input: Some(_)
+  def inverseGuards(x: Option[Int]): Int = x match {
+                                           ^
+t5365b.scala:17: warning: match may not be exhaustive.
+It would fail on the following inputs: None, Some(_)
+  def extractor(x: Option[Int]) = x match {
+                                  ^
+t5365b.scala:20: warning: match may not be exhaustive.
+It would fail on the following inputs: None, Some(_)
+  def repeatedExtractor(x: Option[Int]) = x match {
+                                          ^
+t5365b.scala:23: warning: match may not be exhaustive.
+It would fail on the following input: Some(_)
+  def extractorStrict(x: Option[Int]) = x match {
+                                        ^
+t5365b.scala:27: warning: match may not be exhaustive.
+It would fail on the following input: Some(_)
+  def repeatedExtractorStrict(x: Option[Int]) = x match {
```

Comment:
This warning message is imprecise, it should technically say `Some(x forSome x not in Extractor(x))`.